### PR TITLE
add new templates for azure data share account

### PR DESCRIPTION
**Pull request by dadihe**
# PR Checklist

Check these items before submitting a PR... 

[Contribution Guide](https://github.com/Azure/azure-quickstart-templates/blob/master/1-CONTRIBUTION-GUIDE/README.md)

[Best Practice Guide](https://github.com/Azure/azure-quickstart-templates/blob/master/1-CONTRIBUTION-GUIDE/best-practices.md)

- [x] - Please check this box once you've submitted the PR if you've read through the Contribution Guide and best practices checklist.

## Changelog

*
*
*

### CODE REVIEW

File: 101-data-share-account/README.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
+# Copy data from one folder to another folder in an Azure Blob Storage
```
Comment (bmoore-msft) [inline 101-data-share-account/README.md:1]: this doesn't look correct...

File: 101-data-share-account/README.md [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+
+[![Deploy to Azure](https://raw.githubusercontent.com/Azure/azure-quickstart-templates/master/1-CONTRIBUTION-GUIDE/images/deploytoazure.svg?sanitize=true)](https://portal.azure.com/#create/Microsoft.Template/uri/https%3A%2F%2Fraw.githubusercontent.com%2FAzure%2Fazure-quickstart-templates%2Fmaster%2F101-data-share-account%2Fazuredeploy.json)
+
+[![Deploy to AzureGov](https://raw.githubusercontent.com/Azure/azure-quickstart-templates/master/1-CONTRIBUTION-GUIDE/images/deploytoazuregov.svg?sanitize=true)](https://portal.azure.us/#create/Microsoft.Template/uri/https%3A%2F%2Fraw.githubusercontent.com%2FAzure%2Fazure-quickstart-templates%2Fmaster%2F101-data-share-account%2Fazuredeploy.json)
+
+[![Visualize](https://raw.githubusercontent.com/Azure/azure-quickstart-templates/master/1-CONTRIBUTION-GUIDE/images/visualizebutton.svg?sanitize=true)](http://armviz.io/#/?load=https%3A%2F%2Fraw.githubusercontent.com%2FAzure%2Fazure-quickstart-templates%2Fmaster%2F101-data-share-account%2Fazuredeploy.json)
+
+When you deploy this Azure Resource Manager template, a data share account is created.
```
Comment (bmoore-msft) [inline 101-data-share-account/README.md:18]: what is a data share account?  is there a doc link?

File: 101-data-share-account/azuredeploy.json [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+        }
+    },
+    "variables": {},
+    "resources": [
+        {
+            "type": "Microsoft.DataShare/accounts",
+            "apiVersion": "2019-11-01",
+            "name": "[parameters('accounts_template_name')]",
+            "location": "[parameters('location')]",
+            "identity": {
```
Comment (bmoore-msft) [inline 101-data-share-account/azuredeploy.json:26]: how is this used?  also, the property set doesn't look correct, on a systemAssigned msi the tenant and principal are created by the system, not supplied by the user

File: 101-data-share-account/metadata.json [hunk 4]
```diff
@@ -0,0 +1,22 @@
+{
+  "$schema": "https://aka.ms/azure-quickstart-templates-metadata-schema#",
+  "type": "QuickStart",
+  "itemDisplayName": "Create a data share account",
+  "description": "This template creates a data share account",
+  "summary": "Create a Data Share account",
+  "githubUsername": "dadihe",
+  "dateUpdated": "2020-06-29",
+  "environments":[
```
Comment (bmoore-msft) [inline 101-data-share-account/metadata.json:9]: this isn't correct, see: https://github.com/Azure/azure-quickstart-templates/tree/master/1-CONTRIBUTION-GUIDE#environments

File: 101-data-share-account/README.md (hunk 1 above)
Comment (dadihe) [reply 101-data-share-account/README.md]: fixed

File: 101-data-share-account/README.md (hunk 2 above)
Comment (dadihe) [reply 101-data-share-account/README.md]: https://docs.microsoft.com/en-us/azure/data-share/overview.  I've add this link to the descriptions. should I apply this anywhere else?

Comment (dadihe) [reply 101-data-share-account/README.md]: > https://docs.microsoft.com/en-us/azure/data-share/overview. I've add this link to the descriptions. should I apply this anywhere else?



> what is a data share account? is there a doc link?

https://docs.microsoft.com/en-us/azure/data-share/overview. I've add this link to the descriptions. should I apply this anywhere else?

Comment (dadihe) [reply 101-data-share-account/README.md]: https://docs.microsoft.com/en-us/azure/data-share/overview. I've add this link to the descriptions. should I apply this anywhere else?

File: 101-data-share-account/azuredeploy.json (hunk 3 above)
Comment (dadihe) [reply 101-data-share-account/azuredeploy.json:20]: removed systemassigned msi. user will enter only account name and location to deploy the template.

File: 101-data-share-account/metadata.json (hunk 4 above)
Comment (dadihe) [reply 101-data-share-account/metadata.json]: updated with default value.

Comment (dadihe) [reply 101-data-share-account/metadata.json]: It fails build bc' it is using westus to deploy where we currently don't support. Should I update these issue somewhere? also we have 6 other folders/resource to be updated, can I include them into this PR all should I separate them one each? Thanks

**Comment by dadihe** [comment]
@bmoore-msft Hi Brian, as we discussed I've added a separate parameter file for usgov and corrected inappropriate descriptions. Can you please review again, thanks~

**Pull request merged**